### DISCUSSION TITLE
Quick Links like in XP taskbar

### DISCUSSION THREAD

**Discussion by kungfooman** [linuxmint/discussions#532] [Ideas]
Hi, does anyone know a plugin which adds quick links like in the good old XP taskbar?

Example:

![quicklink](https://github.com/user-attachments/assets/d6adcaf2-3b4e-4523-8503-de2146c2f429)

**Comment by billyswong** [comment]
You will need to get back the "Traditional" layout in Mint 21.1 Cinnamon or before manually
https://forums.linuxmint.com/viewtopic.php?t=405079

**Comment by cplittlfield** [comment]
Not sure what you mean by "guick links".  If you want to be able to launch an app from the panel, find it in the menu, right-click, and select "Add to panel".  For a desktop launcher, click on "Add to desktop".
